src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/help/Add-AzureRmDataLakeAnalyticsFirewallRule.md

```diff
@@ -0,0 +1,153 @@
+---
+external help file: Microsoft.Azure.Commands.DataLakeAnalytics.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# Add-AzureRmDataLakeAnalyticsFirewallRule
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
```

Comment:
@begoldsm it looks like the markdown files added for DataLakeAnalytics have a few sections that contain the templates from platyPS. Would you mind updating these and regenerating the dll-Help.xml file?